src/app/shared/live-region/live-region.component.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  AsyncPipe,
+  NgClass,
+  NgFor,
+} from '@angular/common';
+import {
+  Component,
+  OnInit,
+} from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { LiveRegionService } from './live-region.service';
+
+@Component({
+  selector: `ds-live-region`,
+  templateUrl: './live-region.component.html',
+  styleUrls: ['./live-region.component.scss'],
+  standalone: true,
+  imports: [NgClass, NgFor, AsyncPipe],
+})
+export class LiveRegionComponent implements OnInit {
```

Comment:
Please add a comment or TypeDocs to describe this Component & what it does.  That way other developers can quickly understand this component without having to read the code or track down the original PR which added it.